identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp

```diff
@@ -666,16 +666,6 @@
                                         <%=i18n(recoveryResourceBundle, customText, "sign.up.button")%>
                                     </button>
                                 </div>
-                                <div class="ui divider hidden"></div>
```

Comment:
This'll probably remove the link from other scenarios as well.
Shall we verify?

Reply:
This will only remove it from the complete sign up page..

In the Register page, the Sign in is not removed.

<img width="300" alt="Screenshot 2023-12-05 at 08 24 22" src="https://github.com/wso2/identity-apps/assets/67315176/112ae418-9f3e-4ea0-bb3d-aa141d06368b">

Reply:
@savindi7 shouldn't we check `isPasswordProvisionEnabled` is true and hide the link here?

Reply:
We need to use `skipSignUpEnableCheck`. Will fix this.